creusot/src/backend/term.rs

```diff
@@ -392,7 +392,10 @@ pub(crate) fn lower_literal<'tcx>(
             names.insert(id, subst);
             Exp::Tuple(Vec::new())
         }
-        Literal::Float(f) => Constant::Float(f).into(),
+        Literal::Float(f, fty) => {
+            let _why_ty = floatty_to_ty(names, &fty);
```

Comment:
I needed this to import the module. Not sure if should be propagated down in the `Constant::Float`.